materialize-postgres/sqlgen.go

```diff
@@ -54,6 +54,7 @@ var pgDialect = func() sql.Dialect {
 
 	columnValidator := sql.NewColumnValidator(
 		sql.ColValidation{Types: []string{"bigint"}, Validate: sql.IntegerCompatible},
+		sql.ColValidation{Types: []string{"integer"}, Validate: sql.IntegerCompatible},
```

Comment:
nit: Types is an array -- could this also be a single row of []string{"integer", "bigint"} ?

Reply:
Indeed it can...thanks!